crates/ox_content_ssg/src/html/reader_chrome.css

```
@@ -1,14 +1,21 @@
 .ox-code {
+  --ox-copy-reserved-inline-size: 4.75rem;
   position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)